<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Paster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>粘贴到图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="base_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>要修改的图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paste_image">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>要粘贴的图像</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x_coord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="y_coord">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="base_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>未选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="paste_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>未选择</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="page_start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="page_end">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="x_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X 坐标:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y 坐标:</string>
    </property>
   </widget>
   <widget class="QLabel" name="start_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>页面开始:</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>页面结束:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保持改的图像</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
